app/policies/news_policy.rb

```diff
@@ -0,0 +1,31 @@
+class NewsPolicy < AdminPolicy
```

Comment:
news controller is not under admin namespace

if you need something from there
move it to some other parent policy and inherit both admin policy and news policy from new parent policy